client/packages/coldchain/src/Equipment/DetailView/UploadTab.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { AssetLogPanel } from '../Components';
+
+export const UploadTab = ({ draft, value }: AssetLogPanel) => {
+  return (
+    <AssetLogPanel value={value} draft={draft}>
+      TODO: implement the upload panel
```

Comment:
I wonder about things like this when commiting to develop? Ideally we should hide the tab maybe? I think it's ok at this stage, but as we get closer to a release...

Reply:
good point. have removed the wizard and next button etc, so the modal is now valid.